internal/handlers/web/assets/app.js

```diff
@@ -86,6 +87,16 @@ document.addEventListener("DOMContentLoaded", function () {
     };
   }
 
+  // Function to fetch and display the application version
+  function fetchAppVersion() {
+    fetch('/version')
```

Comment:
It's a good practice to handle potential errors when fetching data. Consider adding error handling for the `fetch('/version')` call in `fetchAppVersion()`. [important]